public/login.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Login page</title>
+
+  <style>
+    body {
+      max-width: 460px;
+      margin: 0 auto;
+      font-family: Arial, Helvetica, sans-serif;
+    }
+
+    form {
+      border: 3px solid #f1f1f1;
+    }
+
+    [type='email'],
+    [type='password'] {
+      box-sizing: border-box;
+      display: inline-block;
+      width: 100%;
+      padding: 12px 20px;
+      margin: 8px 0;
+      border: 1px solid #ccc;
+    }
+
+    button {
+      width: 100%;
+      padding: 14px 20px;
+      margin: 8px 0;
+      color: #fff;
+      cursor: pointer;
+      background-color: #b75400;
+      border: none;
+    }
+
+    button:hover {
+      background-color: #914200;
+    }
+
+
+    .container {
+      padding: 16px;
+    }
+
+  </style>
+</head>
+<body>
+
+<h2>Login page</h2>
+
+<form action="/action_page.php" method="post">
```

Comment:
Change the action to `https://api.core.ac.uk/login`.

@valerapro could we support `multipart/form-data` for this endpoint?